tensoevent/tensoevent.go

```diff
@@ -45,6 +48,23 @@ type GitRepo struct {
 	RemoteURL   string     `json:"remote-url"`
 }
 
+// TFRelease appears in type Event. It describes a terraform run that was executed and it's outcome
+type TFRelease struct {
```

Comment:
I'd rather like this spelled out to make using code clearer:
```suggestion
type TerraformRelease struct {
```
Same for all other instances of `TFSomething`.